tests/test.sh

```diff
@@ -8,8 +8,20 @@ set -e
 SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"  # https://stackoverflow.com/a/246128
 
 if [ -z ${NO_APT_INSTALL+x} ]; then
- sudo DEBIAN_FRONTEND=noninteractive apt-get -qq update
- sudo DEBIAN_FRONTEND=noninteractive apt-get install -y -qq --no-install-recommends openjdk-8-jdk python3 python3-pip docker.io php ruby2.7 dotnet-sdk-6.0
+   if [ "$(uname -m)" = "aarch64" ]; then
```

Comment:
Also missing on aarch64 that I ran were - insallation of `build-essential` and `python3-devel` to build python packages we install and do not have `.whl` files. Please add here under the aarch64 branch.